src/outfit.ts

```diff
@@ -165,6 +166,7 @@ export function tryFillLatte(): boolean {
 
 export function meatOutfit(embezzlerUp: boolean, requirement?: Requirement, sea?: boolean): void {
   const equipMode = embezzlerUp ? "embezzler" : "barf";
+  if (equipMode === "barf") setLocation($location`Barf Mountain`);
```

Comment:
We can also unset this before returning, which might be useful if we are doing any cache compute for whatever reasons.